build/NuGetAdditionalFiles/RunCsc.cmd

```diff
@@ -1,2 +1,4 @@
 echo off
-dotnet %~dp0\csc.dll %*
+if defined DOTNET_HOST_PATH set HOST_PATH=%DOTNET_HOST_PATH%\
+if not defined HOST_PATH set HOST_PATH=
```

Comment:
I suppose this is actually kind of buggy, as it allows leakage of the `HOST_PATH` variable (if `DOTNET_HOST_PATH` is unset and `HOST_PATH` is set, it erroneously uses `HOST_PATH`). Changing it to `if not defined DOTNET_HOST_PATH set HOST_PATH=` would resolve that.